portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,89 @@
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Samiksha Kapoor</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div class='Content'>
+    <div class='Display'>
+      <img class='ProfilePic' src='images/my_photo.jpg'>
+      <div class="Info">
+        <h1>Samiksha Kapoor</h1>
+        <p>
+          <h2>HEY!</h2>
+			Thanks for viewing my portfolio! Let me introduce myself...
+		  <br>
+          I am a Competitive Programmer. I always remain very passionate to learn about
+          new technologies. Loves to solve the Real World Problems and want make a positive impact
+          on this world.I have been a part of lot of communities like Girlscript Patiala, Developer Student Club.
+          Along with that, I am a Microsoft Student partner and a part of WomenTechNetwork.
+		  <br>
+          Also, I am a big MUSIC LOVER.
+        </p>
+	  </div>
+	</div>
+	<p class='GreetingMessage'>Click here to get a message :)</p>
+	<button class="ButtonGreetings" onclick="addRandomGreeting()">Click Here</button>
+	<div class='GreetingMessage' id='greeting-container'></div>
+	<br>
+	<div class='Education'>
+	  <h2 class='Headings'> EDUCATION </h2>
+	  <ul>
+	    <li><b>Thapar Institute of Engineering and Technology,Patiala,Punjab</b></li>
+	    <i>2017-2021</i>
+	    <br>
+        Bachelors of Engineering (BE) in Computers
+		<li><b>DAV Public School,Amritsar,Punjab</b></li>
+		<i >2017</i>
+		<br>
+        Senior Secondary
+		<li><b>DAV Public School,Amritsar,Punjab</b></li>
+		<i >2015</i>
+		<br>
+        Secondary
+      </ul>
+	</div>
+	<h3><b>Check out Some of my work :)</b></h3>
+	<div class='Projects'>
+	  <h2 class='Headings'>PROJECTS</h2>
+	  <ul>
+	  <li><a href="https://github.com/samii9914/PermissionApp.git"><b>GIRLS PERMISSION APP</b></a></li>
+	  A permission App in which girls can take directly the permission for going to library, society work,
+	  late entry request and early leave request without the need for pen and paper.
+	  <li><b>RAILWAY RESERVATION SYSTEM</b></li>
+	  Used different SQL commands and PL SQL procedures and triggers to implement ticket booking for Railways.
+	  <li><a href="https://github.com/samii9914/Guess_The_Number.git"><b>GUESS THE NUMBER GAME</b></a></li>
+	  Web App in which the computer successfully guesses the number entered by the user after some iterations.
+	  </ul>
+	  <p>Currently working on project <b>"SIGN LANGUAGE IDENTIFIER"</b>
+	  <br>
+      The project mainly focus on to convert Indian Sign Language(ISL) to text and further converting speech to
+      Indian Sign Language(ISL).
+      </p>
+	</div>
+	<div class='Experience'>
+	  <h2 class='Headings'> EXPERIENCE </h2>
+	  <ul>
+	    <li><b>Full Stack Intern-</b>Gmetri,Bangalore</li>
+		<i>May,2020-July,2020</i>
+		<li><b>Teaching Assistant(Data Structures and Algorithms,JAVA)-</b>Coding Blocks,New Delhi</li>
+		<i>October,2018-June 2019</i>
+	  </ul>
```

Comment:
Indentation issue
Lines (73, 74, 75) are not sub-tags of <li> but sub-tags of <ul>. These lines should have indentation same as <li> tag in the 2nd line.